plugins/modules/network_settings_workflow_manager.py

```diff
@@ -347,15 +347,42 @@
                     description: Retain existing banner message.
                     type: bool
                 type: dict
+              wired_data_collection:
+                description: Enables or disables the collection of data from wired network devices for telemetry and monitoring purposes.
+                    Applicable from sdk version 2.3.7.6 onwards.
+                suboptions:
+                    enable_wired_data_collection:
+                      description: Enable or disable wired data collection.
+                      type: bool
+                      default: False
+                type: dict
+              wireless_telemetry:
+                description: Enables or disables the collection of telemetry data from wireless network devices for performance monitoring and analysis.
+                    Applicable from sdk version 2.3.7.6 onwards.
+                suboptions:
+                    enable_wired_data_telemetry:
+                      description: Enable or disable wireless telemetry.
+                      type: bool
+                      default: False
+                type: dict
               netflow_collector:
                 description: Netflow collector details under a specific site.
                 suboptions:
+                  collectorType:
```

Comment:
Can you please avoid using Camel case? We have been told multiple times not to use camel case in our workflow manager module. But we are still writing code in camel case.

Reply:
I have reviewed the code and made updates to avoid camel case where possible. If there are any specific instances I missed, please let me know, and I'll address them ASAP. Thank you!